account_credit_control/report/report_credit_control_summary.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_credit_control_summary_document">
+        <t t-call="web.external_layout">
+            <t t-set="doc" t-value="doc.with_context({'lang':doc.partner_id.lang})"/>
+            <t t-set="address">
+                <address t-field="doc.contact_address" t-options="{'widget':'contact', 'fields': ['address', 'name'], 'no_marker': True}"/>
+            </t>
+            <div class="page">
+
+                <h2 id="policy_level">
+                    <span t-field="doc.current_policy_level.name"/>
+                </h2>
+
+                <div id="informations" class="row mt32 mb32">
+                  <div class="col-auto mw-100 mb-2" name="date">
+                    <strong>Date:</strong>
+                    <p class="m-0" t-field="doc.report_date"/>
+                  </div>
+                </div>
+
+                <div class="row mt32 mb32">
+                    <div class="col-xs-12">
+                       <span t-field="doc.current_policy_level.custom_text"/>
+                    </div>
+                </div>
+
+                <h3>Summary</h3>
+
+                <table class="table table-condensed" id="summary_table">
+                    <thead>
+                        <tr>
+                            <th>Invoice number</th>
+                            <th>Invoice date</th>
+                            <th>Date due</th>
+                            <th class="text-right">Invoiced amount</th>
+                            <th class="text-right">Open amount</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr t-foreach="doc.credit_control_line_ids" t-as="l">
+                            <t t-if="l.invoice_id">
+                                <td>
+                                    <span t-field="l.invoice_id.number"/>
```

Comment:
Number is no field, name should be sufficient?

Reply:
@pedrobaeza 
I still get:
```
  File "/Users/brahoo/PycharmProjects/cust/odoo-13.0/odoo/addons/base/models/ir_qweb.py", line 363, in _get_field
    field = record._fields[field_name]
KeyError: 'number'

Error to render compiling AST
KeyError: 'number'
Template: account_credit_control.report_credit_control_summary_document
Path: /t/t/div/table/tbody/tr/t[1]/td/span
Node: <span t-field="l.invoice_id.number"/>
```

I think the line <span t-field="l.invoice_id.number"/> should be removed. Only name is sufficient.